python/common/org/python/stdlib/datetime/TimeDelta.java

```diff
@@ -1,10 +1,34 @@
 package org.python.stdlib.datetime;
 
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.time.Duration;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
+import java.util.Map;
+import java.util.HashMap;
+import java.util.Objects;
 
 public class TimeDelta extends org.python.types.Object {
+    public static final Map<String, org.python.Object> EMPTY_KWARGS = Collections.unmodifiableMap(new HashMap<>());
+    public static final org.python.Object[] EMPTY_ARGS = {};
```

Comment:
These are used as convenient values to put in the constructor (the python and java one) when kwargs or args is not needed. But isn't it better to create more java constructors that accept one args, one kwargs or no arguments? These extra constructors should only be used internally or at testing of course and should not be exposed to python.

It is weird if every class had their own shorthands like these, it would be hard to keep track of them all.

Reply:
Good suggestion, multiple constructors are indeed a cleaner solution. Thanks!